filter-effects-2/Overview.bs

```diff
@@ -50,28 +54,281 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
+5. Draw all of element B, including its background, border, and any children elements, into T’.
+6. If element B has any transforms, effects, or clips, apply those to T’.
+7. Composite the contents of T’ into element B's parent, using source-over compositing.
+
+Note that the effect of the backdrop-filter will not be visible unless some
+portion of element B is semi-transparent. Also note that any opacity applied to
+element B will be applied to the filtered backdrop image as well. Therefore, to
+create a "transparent" element that allows the full filtered backdrop image to
+be seen, you can use "background-color: transparent;".
+
+<pre class=link-defaults>
+  spec:filter-effects-1; type:element; text:filter
+</pre>
+
+The first <a element>filter</a> function or filter reference in the <<filter-value-list>>
+takes the element's <a>Backdrop Root Image</a> as the input image. Subsequent
 operations take the output from the previous filter function or <a
-element>filter</a> reference as the input image. <a element>filter</a> element
+element>filter</a> reference as the input image. The <a element>filter</a> element
 reference functions can specify an alternate input, but still uses the previous
 output as its SourceGraphic.
 
 Filter functions must operate in the sRGB color space.
 
-A computed value of other than ''backdrop-filter/none'' results in the creation of a <a
-href="https://www.w3.org/TR/CSS21/zindex.html">stacking context</a> [[!CSS21]]
-the same way that CSS 'opacity' does. All the elements descendants are rendered
-together as a group with the filter effect applied to the group as a whole.
+Note: A computed value of other than ''backdrop-filter/none'' results in the
+creation of both a <a href="https://www.w3.org/TR/CSS21/zindex.html">stacking
+context</a> [[!CSS21]] and a <a
+href="https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block">Containing
+Block</a> for absolute and fixed position descendants, in the same way that CSS
+'filter' does.
+
+# Backdrop Root # {#BackdropRoot}
+
+The <dfn>Backdrop Root Image</dfn> is defined as the entire painted output produced by
+drawing, in <a
+href="https://www.w3.org/TR/CSS2/zindex.html#painting-order">painting order</a>,
+all content between (and including) the <a>Backdrop Root</a> element and the
+child element that needs access to the "backdrop". No content that is a DOM
+ancestor of the Backdrop Root element should contribute to or affect the
+Backdrop Root Image.
+
+A <dfn>Backdrop Root</dfn> is formed, anywhere in the document, by an element in any of the following scenarios. See [[#BackdropRootTriggers]] for more details on each:
+ * The root element of the document (HTML).
+ * An element with a filter property other than "none".
+ * An element with an opacity value less than 1.
+ * An element with mask, mask-image, mask-border, clip, clip-path properties with values other than “none”.
```

Comment:
I haven't brought this up before, but now that I'm here: How important is it to you that the `clip` property be on this list? In Firefox, the implementation of `clip` is closer to the implementation of rectangle overflow clips than it is to the implementation of `clip-path`. So for us it would be a bit more straightforward to make `clip` not cause backdrop roots, just like overflow clips. I don't care very strongly though, because `clip` is a property that nobody should be using in this day and age.

Reply:
I think we'd be okay removing clips here also. Really, the issue is with anything that gets implemented as a mask (which does include clip path, at least for Chromium). So, done.